docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
       dockerfile: Dockerfile
     volumes:
       - ./:/chroma
```

Comment:
I think we should clean this up and verify that the data is where we expect. The way I think it is working rn is we are mounting your CWD as where the data is stored